src/modules/editor/SoundEditor/Note.ts

```diff
@@ -1,49 +1,52 @@
-
 import * as Tone from "tone";
 import { SampleLibrary } from "@modules/editor/SoundEditor/Tonejs-Instruments";
```

Comment:
(not necessarily related to this line, more of the sample library)

Regarding the overall system, we need to drastically reduce the number of assets that we need for instruments. This will consume a lot of bandwidth in this current state, and we need to find a solution to pitch up and down the same sample instead of having `n` samples per instrument.

I have created a task that would be great to finish off before the end of alpha, but I doubt we'd pull this off at this stage:
https://naucto.atlassian.net/browse/NCTO-114

Not a direct priority, but would be great to take a look and fix.